packages/application-extension/src/index.tsx

```diff
@@ -203,6 +203,7 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
 
     commands.addCommand(CommandIDs.close, {
       label: () => trans.__('Close Tab'),
+      commandDialog: () => trans.__('Close Tab'),
```

Comment:
1. Why would we want to have a new property for this? I think we should reuse `label` and `caption`. This is be related to which shows and discusses how we can customize/caption for different uses https://github.com/jupyterlab/lumino/issues/570 
2. If a new property is really needed (for now I am not convinced), let's think of a new which explains the intent better. the `command dialog` is very unclear for me. Why add `command`? What is `dialog`?

Reply:
I would defer to @gabalafou here who I believe has more experience with screen readers.

Reply:
Hey @gabalafou, do you have any suggestions on how to proceed with this? Thanks in advance.

Reply:
if can you share  some rendered html from your local demo i might be able to help you. i'd need to see the structure of the dom elements you are trying to get announcements from.

Reply:
Hey @tonyfast, thanks for helping out. I have attached a screenshot of the label div with the aria-live region. I am setting the innerHTML of this div when the open/close left sidebar command (ctrl + B) is run. As the text is changing in the aria-live region I would expect the screen reader (MS Narrator) to announce the text but it's not. It announces the keyboard shortcut instead. 

Any feedback or suggestions would be much appreciated. 

![command dialog](https://github.com/jupyterlab/jupyterlab/assets/141649264/ce122614-6a99-457c-97e5-0d5008e8c55e)